ansible/roles/adm/tasks/main.yml

```diff
@@ -39,3 +39,10 @@
 
 - name: sudoers.d/adm
   template: src=sudoers dest=/etc/sudoers.d/adm owner=root group=root mode=0440 validate='visudo -cf %s'
+
+- name: install sudo
+  apt:
+    name: "{{ item }}"
+    state: latest
+  with_items:
+    - "vim"
```

Comment:
I'd suggest to merge it with `install sudo` in the beginning of the file as `apt` module has special fast-path to install all the packages at once.